libraries/Microsoft.Bot.Builder.Ai/LocaleConverterMiddleware.cs

```diff
@@ -27,12 +28,12 @@ public class LocaleConverterMiddleware : IMiddleware
         /// <param name="localeConverter">An ILocaleConverter instance</param>
         public LocaleConverterMiddleware(Func<ITurnContext, string> getUserLocale, Func<ITurnContext, Task<bool>> checkUserLocaleChanged, string toLocale, ILocaleConverter localeConverter)
         {
-            this.localeConverter = localeConverter ?? throw new ArgumentNullException(nameof(localeConverter));
+            this._localeConverter = localeConverter ?? throw new ArgumentNullException(nameof(localeConverter));
```

Comment:
If you're using "_" as a variable prefix, no need to *ALSO* have "this." on there.

Reply:
fixed